### PR TITLE
Add app code for TS, rename precharge sequence

### PR DESCRIPTION
**Pull request by pasitlaothamatas**
### Summary

Quick PR to supplement Edwins precharge PR
- add tractive system object

### Changelist 

- renamed precharge sequence to precharge relay to make the object more descriptive

### Testing Done

not needed?

### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [ ] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [ ] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### CODE REVIEW

File: boards/BMS/Src/main.c [hunk 1]
```diff
@@ -258,20 +260,24 @@ int main(void)
         Io_LTC6813CellVoltages_GetPackVoltage,
         Io_LTC6813CellVoltages_GetAverageCellVoltage);
 
+    ts = App_TractiveSystem_Create(
+        Io_Adc_GetAdc2Channel4Voltage,
```
Comment (timbitss) [inline boards/BMS/Src/main.c:264]: I think the function entered should be `Io_Adc_GetAdc1Channel3Voltage()` for acquiring the differential tractive system voltage

image

Comment (timbitss) [inline boards/BMS/Src/main.c:264]: I think the function entered should be `Io_Adc_GetAdc1Channel3Voltage()` for acquiring the differential tractive system voltage

image

**Review by timbitss — COMMENTED** [review]
just 1 comment, looks good otherwise

File: boards/BMS/Src/main.c (hunk 1 above)
Comment (pasitlaothamatas) [reply boards/BMS/Src/main.c:264]: whoops thanks for the catch

**Review by timbitss — APPROVED** [review]
lgtm

**Pull request merged**